frontend/test/metabase/scenarios/visualizations/pivot_tables.cy.spec.js

```diff
@@ -929,9 +930,39 @@ function assertOnPivotFields() {
 function dragField(startIndex, dropIndex) {
   cy.get(".Icon-grabber2").should("be.visible").as("dragHandle");
 
-  cy.get("@dragHandle").eq(startIndex).trigger("dragstart");
-
-  cy.get("@dragHandle").eq(dropIndex).trigger("drop");
+  const BUTTON_INDEX = 0;
+  const SLOPPY_CLICK_THRESHOLD = 10;
+  cy.get("@dragHandle")
```

Comment:
Had a quick call with Nemanja. Because we're using mouse events and need to determine coordinates, we kinda need to do this nesting (cypress doesn't play nice with async/await). I'm going to leave it like this for now, but in the future we should abstract this out into a more generic helper function and use it in our other DnD tests